ui.tests/test-module/specs/wizard/wizard.runtime.repeatability.spec.js

```diff
@@ -131,3 +131,87 @@ describe("Form with Wizard Container", () => {
 
 
 })
+
+describe('visibility of navigation buttons', function () {
+    const pagePath = "content/forms/af/core-components-it/samples/wizard/navigationWithRepeatability.html";
+    let formContainer = null;
+
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        })
+    })
+
+    const getTabs = () => {
+        return cy.get(".cmp-adaptiveform-wizard__tab");
+    }
+
+    const getWizardPanels = () => {
+        return cy.get(".cmp-adaptiveform-wizard__wizardpanel");
+    }
+
+    const getTabAtIndex = (index) => {
+        return getTabs().eq(index);
+    }
+
+    const getWizardPanelAtIndex = (index) => {
+        return getWizardPanels().eq(index);
+    }
+
+    it("navigating start to end to start", () => {
+        const previousNavButton = '.cmp-adaptiveform-wizard__previousNav',
+            nextNavButton = '.cmp-adaptiveform-wizard__nextNav',
+            wizardItems = formContainer._model.items[0].items;
+
+        const wizardPanelActive = 'cmp-adaptiveform-wizard__wizardpanel--active';
+
+        getWizardPanelAtIndex(0).should('have.class', wizardPanelActive);
+        cy.get(previousNavButton).should('have.attr', 'data-cmp-visible', 'false');
+        for(let i=0; i<6; i++) {
+            cy.get(nextNavButton).click({force: true});
+        };
+
+        getWizardPanelAtIndex(4).should('have.class', wizardPanelActive);
```

Comment:
this might cause flakyness in CI.
what if this statement at line no 174 runs before the click commands are completed